docs/elasticsearch.txt

```diff
@@ -84,6 +84,73 @@ JanusGraph only uses default values for `index-name` and `health-request-timeout
 
 The REST client accepts the `index.[X].bulk-refresh` option. This option controls when changes are made visible to search. See https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-refresh.html[?refresh documentation] for more information.
 
+===== REST Client HTTPS support
```

Comment:
"REST Client HTTPS Configuration". Also this is getting pretty nested ... I'd consider moving up a level (e.g. "=== REST...").

Reply:
moving